gradle/verification-metadata.xml

```diff
@@ -5637,6 +5674,14 @@
             <sha256 value="02824e839f2a2f0e72959fdd30b4897240f05afc43de42d7ba0b18437601c070" origin="Generated by Gradle"/>
          </artifact>
       </component>
+      <component group="org.ow2.asm" name="asm-commons" version="9.6">
+         <artifact name="asm-commons-9.6.jar">
+            <sha256 value="7aefd0d5c0901701c69f7513feda765fb6be33af2ce7aa17c5781fc87657c511" origin="Generated by Gradle"/>
+         </artifact>
+         <artifact name="asm-commons-9.6.pom">
+            <sha256 value="a98ae4895334baf8ff86bd66516210dbd9a03f1a6e15e47dda82afcf6b53d77c" origin="Generated by Gradle"/>
+         </artifact>
+      </component>
       <component group="org.ow2.asm" name="asm-tree" version="9.2">
          <artifact name="asm-tree-9.2.jar">
```

Comment:
Can the old asm-tree entry be removed?

Reply:
apparently they are still referenced ....

Reply:
so both versions are being used?

Reply:
looks like it, I removed it and gradle started complaining about missing metadata.